deepmd/entrypoints/main.py

```diff
@@ -139,7 +145,13 @@ def main_parser() -> argparse.ArgumentParser:
         "train",
         parents=[parser_log, parser_mpi_log],
         help="train a model",
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        formatter_class=RawTextArgumentDefaultsHelpFormatter,
+        epilog=textwrap.dedent("""\
+        examples:
+            dp train input.json
+            dp train input.json -r model.ckpt
+            dp train input.json -i model.ckpt
```

Comment:
use `--restart` and `--init-model` in the examples so they are more self-explanatory?